frontend/public/components/persistent-volume.jsx

```diff
@@ -88,12 +88,46 @@ const PVTableRow = ({obj, index, key, style}) => {
 };
 PVTableRow.displayName = 'PVTableRow';
 
-const Details = ({obj}) => <React.Fragment>
-  <div className="co-m-pane__body">
-    <SectionHeading text="PersistentVolume Overview" />
-    <ResourceSummary resource={obj} podSelector="spec.podSelector" showNodeSelector={false} showPodSelector />
-  </div>
-</React.Fragment>;
+const Details = ({obj: pv}) =>{
+  const storageClassName = _.get(pv, 'spec.storageClassName');
+  const pvcName = _.get(pv, 'spec.claimRef.name');
+  const namespace = _.get(pv, 'spec.claimRef.namespace');
+  const storage = _.get(pv, 'spec.capacity.storage');
+  const accessModes = _.get(pv, 'spec.accessModes');
+  const volumeMode = _.get(pv, 'spec.volumeMode');
+  const reclaimPolicy = _.get(pv, 'spec.persistentVolumeReclaimPolicy');
+  return (
+    <div className="co-m-pane__body">
+      <SectionHeading text="PersistentVolume Overview" />
+      <div className="row">
+        <div className="col-sm-6">
+          <ResourceSummary resource={pv}>
+            <dt>Reclaim Policy</dt>
+            <dd>{reclaimPolicy}</dd>
+          </ResourceSummary>
+        </div>
+        <div className="col-sm-6">
+          <dl>
+            <dt>Status</dt>
+            <dd><PVStatus pv={pv} /></dd>
+            {storage && <React.Fragment><dt>Capacity</dt><dd>{storage}</dd></React.Fragment>}
+            {!_.isEmpty(accessModes) && <React.Fragment><dt>Access Modes</dt><dd>{accessModes.join(', ')}</dd></React.Fragment>}
```

Comment:
You can use the shorthand for `<React.Fragment>Stuff</React.Fragment>` if you want `<>Stuff</>`